scripts/accounts_and_keys.py

```diff
@@ -91,6 +97,10 @@ def convert_accounts_and_keys():
                 cyberway_permission.find({"owner":doc["account"], "name":"active"}).count() or 
                 cyberway_permission.find({"owner":doc["account"], "name":"posting"}).count()):
             continue
+
+        if id_count_permusage - 1 - default_id_count_permusage == int(argv):
```

Comment:
How will this code behave if the order of creating accounts and rights does not match? Are you sure there are no such situations in database?